torchgeo/datasets/chesapeake.py

```diff
@@ -28,23 +28,14 @@
 class Chesapeake(RasterDataset, abc.ABC):
     """Abstract base class for all Chesapeake datasets.
 
-    `Chesapeake Bay High-Resolution Land Cover Project
-    <https://www.chesapeakeconservancy.org/conservation-innovation-center/high-resolution-data/land-cover-data-project/>`_
+    `Chesapeake Bay Program Land Use/Land Cover Data Project
+    <https://www.chesapeakeconservancy.org/conservation-innovation-center/high-resolution-data/lulc-data-project-2022/>`_
     dataset.
 
     This dataset was collected by the Chesapeake Conservancy's Conservation Innovation
     Center (CIC) in partnership with the University of Vermont and WorldView Solutions,
     Inc. It consists of one-meter resolution land cover information for the Chesapeake
     Bay watershed (~100,000 square miles of land).
-
-    For more information, see:
-
-    * `User Guide
-      <https://chesapeakeconservancy.org/wp-content/uploads/2017/01/LandCover101Guide.pdf>`_
-    * `Class Descriptions
-      <https://chesapeakeconservancy.org/wp-content/uploads/2020/03/LC_Class_Descriptions.pdf>`_
-    * `Accuracy Assessment
-      <https://chesapeakeconservancy.org/wp-content/uploads/2017/01/Chesapeake_Conservancy_Accuracy_Assessment_Methodology.pdf>`_
```

Comment:
Some of these links are broken. Most of these docs are linked on the main dataset page, so I don't think we need to explicitly reference them.

Reply:
Fine with me